packages/functions-utils/README.md

```diff
@@ -0,0 +1,33 @@
+<img src="../../static/logo.png" width="400"/><br>
+
+[![Coverage Status](https://codecov.io/gh/netlify/build/branch/master/graph/badge.svg)](https://codecov.io/gh/netlify/build)
+[![Build](https://github.com/netlify/build/workflows/Build/badge.svg)](https://github.com/netlify/build/actions)
+
+Utility for adding Functions files in Netlify Build.
```

Comment:
worth explaining why you might want to use this?

Reply:
Yes, you're right! I'm going to improve the `README` in a follow-up PR.

Reply:
Done at #645.